Dockerfile

```diff
@@ -135,6 +135,8 @@ RUN apt-get -y install zlib1g-dev liblcms2-dev libwebp-dev libgeos-dev && \
     pip install cartopy && \
     # MXNet
     pip install mxnet && \
+    # Uninstall any old numpy
+    pip uninstall numpy; \
```

Comment:
for consistency, I would look replace the `;` by `&&`.

Reply:
I was doing that in case `numpy` wasn't installed before, but I can change it if you prefer?

Reply:
I think with the `;`, if any command before (like the pip install mxnet) fails, then the ones after can still succceed and the overall RUN command will succeed.

This means, we could have a successful build but w/o some packages installed.